ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<blockquote>Create, train, and deploy a text classification model on Vertex AI</blockquote><p>Suggest specifying as "text classification AutoML model" since Vertex AI can also include custom models</p><p> </p><blockquote>Train a model</blockquote><p>Suggest "Train an AutoML model"</p><p> </p><p>If possible, also suggest describing the sample data and the end goal in the intro. Something like, "This sample is using the Happy Moments sample dataset to train a model that can be use to classify text. The resulting model classifies happy moments into categories reflecting the causes of happiness."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done